io.github.bruttazz.FirefoxDevFlatpak.yml

```diff
@@ -0,0 +1,92 @@
+id: io.github.bruttazz.FirefoxDevFlatpak
```

Comment:
this app id is unlikely to be acceptable for a Firefox package. This is clearly not "under the control of upstream developers".

Reply:
Can you suggest which, base I should give. I mean, I'm not sure whether or not I can use their domain